source/about/cloud-subscriptions.rst

```diff
@@ -111,10 +111,15 @@ What happens if I decide to purchase a subscription to Mattermost Enterprise dur
 
 Your plan immediately changes to the plan you've upgraded to. Your trial will continue and on day 31 billing will commence. You won't incur any charges of your new plan until the trial period is over. Depending on what day of the month this is, you'll be billed prorata. You'll receive your bill at the end of the calendar month for this month and going forward.
 
-How do I change my subscription from Mattermost Enterprise to Mattermost Professional?
+How do I change my subscription to Cloud Free?
```

Comment:
I tried to make this more generic but it's possible the original meaning has been lost. Let me know what you think.